test/functionalTest/cases/0000_cli/command_line_options.test

```diff
@@ -62,5 +62,6 @@ Usage: contextBroker  [option '-U' (extended usage)]
                       [option '-writeConcern' <db write concern (0:unacknowledged, 1:acknowledged)>]
                       [option '-corsOrigin' <CORS allowed origin. use '__ALL' for any>]
                       [option '-cprForwardLimit' <maximum number of forwarded requests to Context Providers for a single client request>]
-                      
+                      [option '-subCacheIval' <interval in seconds between calls to Subscription Cache refresh>]
```

Comment:
The .md which enumrates Orion CLIs should be modified to add this. Remember that PRs now include modifications not only to code and test, but also to docs (either in .md and .apib, when applicable).